data/card_definitions.json

```diff
@@ -186,6 +188,8 @@
         ],
         "collab_effects": [
             {
+                "proxy_full_name": "hSD01-004_PROXY_FULL_NAME_1",
```

Comment:
I'm not thrilled about adding full text for effects that we should be able to pretty reasonably format nicely programmatically.